docs/get-started.md

```diff
@@ -11,7 +11,7 @@ You can click on the following link to generate a repository using the template:
 In both cases, the next steps are:
 
 1. Enter a name for your new repository.
-1. Important: Check "Include all branches".
+1. **Important:** Check "Include all branches".
```

Comment:
I made this bold because I missed this while going sequentially down the list and executing each item. Since I am not the org owner this required I reached out to the admin, had them delete the repository and then forked the template again. I think this should help prevent people lose time at this step in the future. Maybe we can even reword this and add "Do not save yet." after the first step?